package.json

```diff
@@ -10,7 +10,7 @@
     "dist/*"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "vitest",
```

Comment:
Hi, @ad1992 have you tried running the test locally? The `coverage` functionality is already built-in to `"test": "vitest",` in case you are not aware.

For your convenience, please refer to the `vitest.config.ts` file where:

```Typescript
...
    // Enable coverage generation and reporting via the
    // @vitest/coverage-v8 dev dependency.
    coverage: {
      enabled: true,
      include: ["src/**/*.ts"],
      provider: "v8",

      // Json and json-summary are required for GitHub vitest coverage report.
      // For better debugging, enable generate coverage even on failure.
      reporter: ["text", "html", "clover", "json", "json-summary"],
      reportOnFailure: true,

      reportsDirectory: "coverage",
      thresholds: {
        statements: 90,
        branches: 90,
        functions: 90,
        lines: 90,
      }
    },
...
```

Reply:
- [x] Change rejected due to already implemented first in dea26e8.